action.py

```diff
@@ -0,0 +1,28 @@
+import sqlite3
+import sys
+
+import printdb
+from DAO import _Activities, _Products
+from DTO import *
+
+
+def main(actions_file):
+    db_conn = sqlite3.connect(DB_FILE_NAME)
+    db_cursor = db_conn.cursor()
+    with open(actions_file, 'r') as f:
+        for line in f.readlines():
+            args = line.split(',')
+            args = [arg.strip() for arg in args]
+            activity = Activity(*args)
+            curr_quantity = _Products(db_cursor).get_quantity(activity.product_id)
+            new_quantity = curr_quantity + int(activity.quantity)
+            if new_quantity > 0:
```

Comment:
Change the > to >= to accomodate for exact removals (do you also need to remove it if the new quantity is exactly 0?)

Reply:
Absolutely right, I'll fix it